Library/ViewModels/ShippingRuleCellViewModel.swift

```diff
@@ -28,8 +28,7 @@ public final class ShippingRuleCellViewModel: ShippingRuleCellViewModelType,
       .map { $0.selectedShippingRule == $0.shippingRule }
 
     self.textLabelText = data
-      .map { ($0.project, $0.shippingRule) }
-      .map(formattedValue(_:shippingRule:))
+      .map(\.shippingRule.location.localizedName)
```

Comment:
Just to make sure, we don't want to show the shipping cost for any reward / add-on?

Reply:
Yeah that's the decision that we reached. We figured backers are unlikely to ship to a different location based on cost, they would just ship to where they live.